quickstarts/hello-rules-multi-module/README.md

```diff
@@ -1,10 +1,14 @@
 ## Red Hat Decision Manager Kie Server Quickstart - Multi module
 
-This quickstart is intend to be used with the [RHDM Kie Server](https://github.com/jboss-container-images/rhdm-7-openshift-image/tree/master/kieserver) image.
+This quickstart is intend to be used with the [RHDM Kie Server](https://github.com/jboss-container-images/rhdm-7-openshift-image/tree/7.8.1.GA/kieserver) image.
```

Comment:
On the master branch we probably do not want to point to release branch of Kie Server, right?